tensorflow/core/framework/kernel_def.proto

```diff
@@ -16,6 +16,12 @@ message KernelDef {
 
   // Type of device this kernel runs on.
   string device_type = 2;
+ 
+  // Sub-type of device this kernel runs on.
+  // For first-party devices, subdevice_type is the same as device type.
```

Comment:
Would it be more natural to leave subdevice_type as the empty string for first party devices?

Reply:
we are ok if the subdevice_type is empty for the first party devices, the issue is we don't know whether the string generated in kernel_def.proto is always empty by default? as it will be the key of kernel registration and dispatch.

Reply:
done.